scripts/installer/install.sh

```diff
@@ -401,6 +401,15 @@ if [ -n "$onl_platformlib" ]; then
     ln -s "$onl_platformlib" "$bisdn_linux_mnt/usr/lib/libonlp-platform.so.1"
 fi
 
+# setup hostname
+if [ ! -f "$bisdn_linux_mnt/etc/hostname" ]; then
+    # use ONIE machine name and replace underscores with dashes, as
+    # underscores are not allowed
+    hostname="$(onie-sysinfo -m | tr '_' '-')"
+    echo $hostname > "$bisdn_linux_mnt/etc/hostname"
+    echo "127.0.0.1 $hostname" >> "$bisdn_linux_mnt/etc/hosts"
```

Comment:
where is this file created initially? should we maybe modify the original one instead of appending it?

Reply:
This is mirroring what yocto's base-files does if the hostname is set, see https://git.yoctoproject.org/poky/tree/meta/recipes-core/base-files/base-files_3.0.14.bb?h=kirkstone#n135

Reply:
But I just saw I copied it badly :P

Reply:
Fixed it-